TS02-Convert-to-Date-Time-Class-POSIX.Rmd

```diff
@@ -86,22 +87,12 @@ library(lubridate)  #work with dates
 #set working directory to ensure R can find the file we wish to import
 #setwd("working-dir-path-here")
 
```

Comment:
Just to clarify prior to accepting pull request @mguarinello ,  did importing the data get back into this lesson?